FuelSDK/objects.py

```diff
@@ -140,6 +140,11 @@ def __init__(self):
         super(ET_Email, self).__init__()
         self.obj_type = 'Email'
 
+class ET_EmailSendDefinition(ET_CUDSupport):
```

Comment:
how do you use this? what scope would you set for retrieving email send notifications?